subsys/bluetooth/audio/bap_broadcast_assistant.c

```diff
@@ -85,6 +85,58 @@ static int16_t lookup_index_by_handle(uint16_t handle)
 	return -1;
 }
 
+static void bap_broadcast_assistant_discover_complete(struct bt_conn *conn, int err,
+						      uint8_t recv_state_count)
+{
+	struct bt_bap_broadcast_assistant_cb *listener, *next;
+
+	SYS_SLIST_FOR_EACH_CONTAINER_SAFE(&broadcast_assistant_cbs,
+					  listener, next, _node) {
+		if (listener->discover) {
+			listener->discover(conn, err, recv_state_count);
+		}
+	}
+}
+
+static void bap_broadcast_assistant_recv_state_changed(struct bt_conn *conn, int err,
+						       const struct bt_bap_scan_delegator_recv_state
+						       *state)
```

Comment:
```suggestion
static void
bap_broadcast_assistant_recv_state_changed(struct bt_conn *conn, int err,
					   const struct bt_bap_scan_delegator_recv_state *state)
```

or
```suggestion
static void bap_broadcast_assistant_recv_state_changed(
	struct bt_conn *conn, int err, const struct bt_bap_scan_delegator_recv_state *state)
```
would be better than adding a newline between the type and variable (consider using clang-format to help format)